protocol/xmrmaker/net.go

```diff
@@ -51,8 +51,9 @@ func (inst *Instance) initiate(
 		}
 	}
 
-	// checks passed, delete the offer for now
-	if err = inst.offerManager.DeleteOffer(offer.ID); err != nil {
+	// checks passed, delete the offer from memory for now
+	_, _, err = inst.offerManager.TakeOffer(offer.ID)
```

Comment:
It looks like `TakeOffer` doesn't document the subtle nuance that it removes the entry from the cache while leaving it in the database.  Can we also add that directly on the `TakeOffer` method?